ci-operator/step-registry/multiarch/tuning-operator/OWNERS

```diff
@@ -0,0 +1,4 @@
+approvers:
```

Comment:
add yourself here to have the `approved` by default

Reply:
is there a reason why you wouldn't add yourself to the multiarch-approvers and multiarch-reviewers group directly?

Reply:
we will have a fix of this issue in another pr.